packages/config/config/devices/0x0234/zso7300.json

```diff
@@ -0,0 +1,109 @@
+{
+	"manufacturer": "Logic Group",
+	"manufacturerId": "0x0234",
+	"label": "ZSO7300",
+	"description": "Wall Socket",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x012a"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Relay (Basic Report)",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Relay (Binary Switch Set)",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "Meter Report Time",
+			"description": "Values 1-127 = seconds; 128-255 = minutes (minus 127)",
+			"defaultValue": 60
+		},
+		"2": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Button 2 Controls Relay 1"
+		},
+		"3": {
+			"$import": "templates/logic_group_template.json#watt_threshold_change"
+		},
+		"4": {
+			"$import": "templates/logic_group_template.json#power_overload_limit"
+		},
+		"5": {
+			"$import": "templates/logic_group_template.json#led_power_consumption_mode"
+		},
+		"6[0xff000000]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "LED Indicator (On): Red",
+			"valueSize": 4,
+			"defaultValue": 0
+		},
+		"6[0xff0000]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "LED Indicator (On): Green",
+			"valueSize": 4,
+			"defaultValue": 255
+		},
+		"6[0xff00]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "LED Indicator (On): Blue",
+			"valueSize": 4,
+			"defaultValue": 0
+		},
+		"7[0xff000000]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "LED Indicator (Off): Red",
+			"valueSize": 4,
+			"defaultValue": 143
+		},
+		"7[0xff0000]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "LED Indicator (Off): Green",
+			"valueSize": 4,
+			"defaultValue": 0
+		},
+		"7[0xff00]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "LED Indicator (Off): Blue",
+			"valueSize": 4,
+			"defaultValue": 255
+		},
```

Comment:
The import `base_enable_disable` doesn't make sense here. It is a toggle for `Enable (1)` / `Disable (0)`.
I must have overlooked that last time.